charts/flagsmith/templates/jobs-migrate-db.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ template "flagsmith.fullname" . }}-migrate-db-{{ .Release.Revision }}-{{ randAlphaNum 5 }}
+  name: {{ template "flagsmith.fullname" . }}-migrate-db-{{ .Release.Revision }}-{{ randAlphaNum 5 | lower }}
```

Comment:
Why do we need to do this?

Reply:
K8S does not accept uppercase characters 😅